completions/ip

```diff
@@ -318,6 +318,24 @@ _ip()
             esac
             ;;
 
+        netns)
+            case $subcmd in
+                add|identify|list|list-id|monitor)
+                    # TODO
+                    ;;
+                delete|exec|pids|set)
+                    [[ $prev == $subcmd ]] && \
+                        COMPREPLY=( $( compgen -W '$( command \
```

Comment:
Cosmetic: no space after the paren in `$(` for consistency, ditto for the closing paren and similarly below.